config/environments/production.rb

```diff
@@ -61,8 +61,8 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment).
-  config.active_job.queue_adapter = :sidekiq
```

Comment:
Does this need to be retained?

Reply:
i would assume so... I missed this, it was removed by the app:update script, so I added back in